spec/support/unit/helpers/rails_versions.rb

```diff
@@ -38,5 +38,13 @@ def rails_lt_5?
     def rails_5_x?
       rails_version =~ '~> 5.0'
     end
+
+    def rails_lt_6?
```

Comment:
Good thinking, but what if we add these when we need them?

Reply:
Yeah, let's go ahead and leave them out for now, and we can add them when it comes time.

Reply:
Have removed